resources/instructions/createSObjectLwcQuickActions.html

```diff
@@ -56,54 +56,87 @@ <h1>Create sObject LWC Quick Actions</h1>
             </tbody>
         </table>
 
-        <label
-            style="display: block; margin-top: 10px; margin-bottom: 8px"
-            for="generateLwcQuickActionsButton"
-        >
-            Click 'Generate LWC Quick Actions' to create the missing LWC Quick
-            Actions:
-        </label>
-        <button id="generateLwcQuickActionsButton">
-            Generate LWC Quick Actions
-        </button>
-        <script>
-            const generateQuickActionsButtonElement = document.getElementById(
+    <label style="display: block; margin-top: 10px; margin-bottom: 8px" for="generateLwcQuickActionsButton">
+        Click 'Generate LWC Quick Actions' to create the missing LWC Quick
+        Actions:
+    </label>
+
+    <button id="generateLwcQuickActionsButton">
+        Generate LWC Quick Actions
+    </button>
+
+    <button id="continueButton">
+        Continue
+    </button>
+
+    <script>
+        const generateQuickActionsButtonElement = document.getElementById(
+            'generateLwcQuickActionsButton'
+        );
+
+        const continueButtonElement = document.getElementById(
+            'continueButton'
+        );
+
+        const quickActionStatusTable = document.getElementById(
+            'quickActionStatusTable'
+        );
+
+        generateQuickActionsButtonElement.addEventListener('click', () => {
+            webviewMessaging.sendMessageRequest(
+                'generateLwcQuickActionsButton',
```

Comment:
Nit: these requests might benefit from more descriptive names of the _action_ requested, rather than the _element_ initiating the action. Same for the `continueButtonElement` click event below.

Reply:
Good idea, will rename